### PR TITLE
[DOCFIX] Update User-Cli Docs

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

**Comment by bradyoo** [comment]
@ns1123

File: docs/en/operation/User-CLI.md [hunk 1]
```diff
@@ -584,9 +584,19 @@ For example, `cp` can be used to copy files between under storage systems.
...
+If the source designates a directory, `distributedCp` copies the entire subtree at source to the destination.
+
+```console
+$ ./bin/alluxio fs distributedCp /data/1023 /data/1024
+```
+
 ### distributedLoad
 
-The `distributedLoad` command moves a file or directory from the under storage system into Alluxio storage in distributed 
+The `distributedLoad` command moves a file or directory from the under storage system into Alluxio storage distributed
```
Comment (ns1123) [inline docs/en/operation/User-CLI.md]: Can we use a better word than "moves"? The implication being the data is then no longer in the under storage system. Maybe just "load"?

File: docs/en/operation/User-CLI.md [hunk 2]
```diff
@@ -597,6 +607,16 @@ The `--replication` flag can be used to load the data into multiple workers.
 $ ./bin/alluxio fs distributedLoad --replication 2 /data/today
 ```
 
+### distributedMv
+
+The `distributedMv` command moves a file or directory in the Alluxio file system distributed across workers.
```
Comment (ns1123) [inline docs/en/operation/User-CLI.md]: If it's just one file is it still distributed somehow? Or does it just fall back to using a regular move?

File: docs/en/operation/User-CLI.md [hunk 3]
```diff
@@ -597,6 +607,16 @@ The `--replication` flag can be used to load the data into multiple workers.
...
+
+If the source designates a directory, `distributedMv` moves the entire subtree at source to the destination.
```
Comment (ns1123) [inline docs/en/operation/User-CLI.md]: Does distributed move work across mount points? If not, we should mention that here.

File: docs/en/operation/User-CLI.md (hunk 2 above)
Comment (bradyoo) [reply docs/en/operation/User-CLI.md]: I should say it uses the JobService here. (No fallback)

File: docs/en/operation/User-CLI.md (hunk 3 above)
Comment (bradyoo) [reply docs/en/operation/User-CLI.md]: I don't actually know but I believe it does.

**Comment by bradyoo** [comment]
reflected on the feedback but I don't know if this is what you wanted. @ns1123

File: docs/en/operation/User-CLI.md (hunk 3 above)
Comment (ns1123) [reply docs/en/operation/User-CLI.md]: Based on the code it looks like it does a copy and a delete which should be fine.

File: docs/en/operation/User-CLI.md [hunk 4]
```diff
@@ -310,9 +310,9 @@ $ ./bin/alluxio copyDir conf/alluxio-site.properties
 
 The `clearCache` command drops the OS buffer cache.
 
-### confDocGen
+### docGen
 
-The `confDocGen` autogenerates configuration documentation based on the current source code.
+The `docGen` autogenerates documentation based on the current source code.
```
Comment (ZacBlanco) [inline docs/en/operation/User-CLI.md]:
```suggestion
The `docGen` command autogenerates documentation based on the current source code.
```

**Comment by gpang** [comment]
alluxio-bot, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by ZacBlanco** [comment]
jenkins, test this please

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**